.eslintrc

```diff
@@ -1,106 +1,77 @@
 {
-  "extends": [
-    "eslint:recommended",
-    "plugin:prettier/recommended"
-  ],
-  "rules": {
-    "curly": [
-      "error",
-      "all"
-    ],
-    "eqeqeq": [
-      "error",
-      "smart"
-    ],
-    "import/no-extraneous-dependencies": [
-      "error",
-      {
-        "devDependencies": true,
-        "optionalDependencies": false,
-        "peerDependencies": false
-      }
-    ],
-    "no-shadow": [
-      "error",
-      {
-        "hoist": "all"
-      }
-    ],
-    "prefer-const": "error",
-    "import/order": [
-      "error",
-      {
-        "groups": [
-          [
-            "external",
-            "builtin"
-          ],
-          "internal",
-          [
-            "parent",
-            "sibling",
-            "index"
-          ]
+    "extends": ["eslint:recommended", "plugin:prettier/recommended"],
```

Comment:
Can you confirm that there are no other changes in this file than code formatting?

Reply:
Yes, I only formatted the file with `prettier`